tests/models/mistral3/test_modeling_mistral3.py

```diff
@@ -361,8 +361,8 @@ def test_mistral3_integration_generate(self):
         expected_outputs = Expectations(
             {
                 ("xpu", 3): "The image features two cats resting on a pink blanket. The cat on the left is a kitten",
-                ("cuda", 7): "The image features two cats resting on a pink blanket. The cat on the left is a kitten",
-                ("cuda", 8): "The image features two cats resting on a pink blanket. The cat on the left is a small kit",
+                ("cuda", 7): 'The image features two tabby cats lying on a pink surface, which appears to be a couch or',
+                ("cuda", 8): 'The image features two cats lying on a pink surface, which appears to be a couch or a bed',
```

Comment:
In #37882   These values are not updated, but I remembered I checked them on CI runners.
From CI reports, it looks like we get  different outputs every day, but I can't reproduce that: I ran them several times and even trigger via workflow runs, all give the same results now (and passing with the updated values).

 TODO: monitor in the next few days to see if we get them passing.

Reply:
well, I again different values today v.s. the one I get yesterday, wtf .... 😭

Reply:
the order of tests here is the same (unless we add , remove or skip some tests).

It seems to me that, only if I rebuild the docker image, we will see the outputs change. If I use the same docker image built, no matter how I run (manually in SSH runners or trigger by GitHub Actions), they all give the same outputs.

I am still checking

Reply:
@zucchini-nlp if you ever want to hear this sad sorry (to me)

https://huggingface.slack.com/archives/C01NE71C4F7/p1750885359361689